jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -1,104 +1,121 @@
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
 
-## Introduction
-{: #introduction }
+TIP: Config policy management is currently in **open preview**. All aspects of the feature are subject to change. **We recommend you do not use this feature in production**.
 
-CircleCI uses `config.yml` files to define CI pipelines at the project level. For most use cases this is a convenient way of developing and iterating quickly as each pipeline can be made to meet the needs of the project as it grows. However, from an organizational standpoint, it can be tricky to manage and enforce organization-wide conventions and security policies. 
+== Introduction
 
-Config Policy Management uses a decision engine leveraging OPA (Open-Policy-Agent) to allow users to specify policies, and return a decision about if a Pipeline's Config complies with those policies. In the strictest case, if a pipeline configuration does not comply with the organization's policies, that pipeline will be blocked from running until it does comply. Decision's are also stored and can be audited allowing for useful feedback about what pipeline defintions are being run in your organization.
+Config policy management allows you to create policies for governing project configurations. In its strictest case, if a project configuration does not comply with the rules set out in the associated policy, that pipeline will be blocked from being triggered until it does comply.
 
-Policies are written in the `rego` query language defined by OPA, and following certain circleci conventions / semantics, rule violations are surfaced when pipelines are triggered.
+CircleCI uses `config.yml` files to define CI pipelines at the project level. For most use cases this is a convenient way of developing and iterating quickly as each pipeline can be made to meet the needs of the project as it grows. However, from an organizational standpoint, it can be tricky to manage and enforce organization-wide conventions and security policies. Config policy management adds in this layer of control.
 
-The following sections will describe the CircleCI specific semantics, what is a CircleCI Decision, and how to use the CircleCI-CLI to manage your policies.
+Config policy decisions are stored and can be audited. This provides useful data about the pipeline defintions being run within your organization.
 
-#TODO: A section on the architecture/process - how a config is evaluated 
+== How config policy management works
 
-## Writing Rego Policies using CircleCI Domain-Specific Language
-{: #writing-rego-policies }
+Config policy management uses a decision engine based on link:https://www.openpolicyagent.org/[Open Policy Agent (OPA)]. Policies are written in the `rego` query language, as defined by OPA. Rule violations are surfaced when pipelines are triggered.
 
-# TODO: re-read to ensure no confusion between rego rules and CircleCI violations
+#TODO: A section on the architecture/process - how a config is evaluated#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
This `todo` is not critical for now, and can be removed and deferred.